Clinical-Genomics/patientMatcher
# Unblock pymongo, pytest and mongomock dependencies

### This PR adds | fixes:
- Do not freeze pymongo, pytest and mongomock dependencies. Fix #142 and fix #143 

**How to prepare for test**:
- Install on clinical-db stage

### How to test:
- Save patient data from Scout interface, check that adding a new patient with one or more variants works.
- Send a request for stats from the command line (curl) to get the server stats to make sure the info endpoint still works

### Expected outcome:
- The above tests should work

### Review:
- [x] Tests executed by CR, Travis CI

This [version](https://semver.org/) is a:
